client/rpc/Cargo.toml

```diff
@@ -4,7 +4,7 @@ description = "Ethereum RPC (web3) compatibility layer for Darwinia."
 edition     = "2021"
 homepage    = "https://darwinia.network"
 license     = "GPL-3.0"
-name        = "dc-rpc"
+name        = "darwinia-rpc"
```

Comment:
This name is not good.

https://github.com/darwinia-network/darwinia/blob/main/rpc/Cargo.toml#L7

Reply:
Make sense, I will update this later.

Reply:
Something like `darwinia-client-rpc`

Reply:
@AurevoirXavier Take a review again.